### PR TITLE
Feature/milestone2:Bookstore-cms

### PR DESCRIPTION
**Pull request by DeliceLydia**
# Description

### In this second milestone of the bookstore-cms I :

- Prepared the directory structure
- Implemented app component
- Implemented booklist component
- Implemented bookform component
- Implemented book component
- Defined redux actions and reducers
- Initialized redux store

Thanks!

### CODE REVIEW

File: src/Components/Book.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/destructuring-assignment */
```
Comment (bellom) [inline src/Components/Book.js]: Inline comment: 
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure Eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

File: src/Containers/BookList.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```
Comment (bellom) [inline src/Containers/BookList.js]: Inline comment: 
With the ESLint inline comment, you are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

_Hint: I think if you add the prop-types, you won't need to use the inline comment._

File: src/Containers/BookList.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            ))
+            : (
+              <tr>
+                <td>No books!</td>
+              </tr>
+            )
+      }
+
+      </tbody>
+    </table>
```
Comment (bellom) [inline src/Containers/BookList.js]: Kindly fix the empty line and indentation.

Hint: If on VSCode, highlight your entire code and right-click and click Format document.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @DeliceLydia  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi Delice :slightly_smiling_face: 

We are good here, go on with the next milestone.

### Happy coding :man_technologist: 


#### ``` Uchechukwu Henry Inyama ```

**Pull request merged**